app/models/Property.test.ts

```diff
@@ -1,56 +1,183 @@
 import { Property } from "./Property";
+import { HOUSE_BREAKDOWN_PERCENTAGES, MAINTENANCE_LEVELS } from './constants';
 
-let property: Property;
-
-beforeEach(() => {
-  property = new Property({
-    postcode: "WV8 1HG",
-    houseType: "T",
-    numberOfBedrooms: 2,
-    age: 10,
-    size: 88,
-    maintenancePercentage: 0.02,
-    newBuildPricePerMetre: 2120,
-    averageMarketPrice: 218091.58,
-    itl3: "TLG24",
-  });
-});
-
-it("can be instantiated", () => {
-  expect(property).toBeInstanceOf(Property);
-});
-
-it("correctly calculates the newBuildPrice", () => {
-  expect(property.newBuildPrice).toBeCloseTo(186560);
-});
-
-it("correctly calculates the depreciatedBuildPrice", () => {
-  expect(property.depreciatedBuildPrice).toBeCloseTo(110717.45);
-});
-
-it("correctly calculates the bedWeightedAveragePrice", () => {
-  expect(property.bedWeightedAveragePrice).toBeCloseTo(218091.58);
-});
-
-it("correctly calculates the landPrice", () => {
-  expect(property.landPrice).toBeCloseTo(31531.579);
-});
-
-it("correctly calculates the landToTotalRatio", () => {
-  expect(property.landToTotalRatio).toBeCloseTo(0.14);
-});
-
-it("correctly calculates the values even for number of bedroooms exceeding the max ", () => {
-  property = new Property({
-    postcode: "WV8 1HG",
-    houseType: "T",
-    numberOfBedrooms: 20,
-    age: 9,
-    size: 88,
-    maintenancePercentage: 0.02,
-    newBuildPricePerMetre: 2120,
-    averageMarketPrice: 218091.58,
-    itl3: "TLG24",
-  });
-  expect(property).toBeInstanceOf(Property);
-});
+describe('Property', () => {
+  beforeEach(() => {
+    property = new Property({
```

Comment:
I think my thinking was that we would need to re-instantiate after tests changed the parameters of the object, eg in the test lines 41-53:
``` 
   property = new Property({
      postcode: "WV8 1HG",
      houseType: "T",
      numberOfBedrooms: 20,
      age: 11,
      size: 88,
      maintenancePercentage: 0.02,
      newBuildPricePerMetre: 2120,
      averageMarketPrice: 218091.58,
      itl3: "TLG24",
    });
```

However, I think it might be good to rewrite all tests after `oz/test-helpers` (#151) is merged. Is this worth fixing here?